__tests__/utils/env-helpers.test.ts

```
@@ -0,0 +1,45 @@
+
+  it('uses NEXT_PUBLIC_SITE_URL on server side', () => {
+    // @ts-ignore
+    delete global.window
+    process.env.NEXT_PUBLIC_SITE_URL = 'https://site.example.com'
+    const url = getAppUrl()
+    expect(url).toBe('https://site.example.com')
+  })
+
+  it('falls back to localhost when no env vars are set', () => {
+    // @ts-ignore
+    delete global.window
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)